core/src/main.jl

```diff
@@ -57,8 +57,11 @@ function main(ARGS::Vector{String})::Cint
             logger =
                 Ribasim.setup_logger(; verbosity = config.logging.verbosity, stream = io)
             with_logger(logger) do
-                ribasim_version = pkgversion(Ribasim)
+                ribasim_version = string(pkgversion(Ribasim))
                 (; starttime, endtime) = config
+                if string(ribasim_version) != config.ribasim_version
+                    @warn "The Ribasim version in the TOML config file does not match the used Ribasim CLI version." config.ribasim_version ribasim_version
+                end
```

Comment:
Maybe make it a bit easier to see which one is the version in the config and which one in the binary?

Reply:
Is this better? I added the `cli.` In general I like this type of logging that splits the values from the message.

```
┌ Warning: The Ribasim version in the TOML config file does not match the used Ribasim CLI version.
│   config.ribasim_version = "a_different_version"
│   cli.ribasim_version = "2024.2.0"
└ @ Ribasim d:\Ribasim\core\src\main.jl:63
```